examples/rendering-elements/update-rendered-element.js

```diff
@@ -1,8 +1,8 @@
 function tick() {
   const element = (
     <div>
-      <h1>Hello, world!</h1>
-      <h2>It is {new Date().toLocaleTimeString()}.</h2>
+      <h1>Привіт, світе</h1>
```

Comment:
@klymenkoo suggested to keep it as `Hello, world`

Reply:
@ApolllonDev are we going to continue `examples/*` folder snippet translations?